list.md

```diff
@@ -77,3 +77,4 @@ To add your project to the list follow the instructions [here](/README.md)
 | EasyGoods | A commerce project for pi commerce hackathon | Consumer utilities / Business tools | Javascript/NextJS, FireBase        | [EasyGoods](https://github.com/0205miss/EasyGoods)             |     N/A                      | [EasyGoods](https://easygoods5604.pinet.com)        | (When available)        |
 | RadioForUs | The first app in Pi ecosystem to listen radio stations around the World, from Pi Browser. Pi Hackathon winner at August, 2023. | Music, Entertainment | Odoo, PHP, HTML, JavaScript, Wordpress | [radio-for-us](https://github.com/pi-apps/radio-for-us)  | [pi://brainstorm.pi/project/63e3bbc54c61497c34ced556](pi://brainstorm.pi/project/63e3bbc54c61497c34ced556) | [https://radioforus.com](https://radioforus.com) | [https://mainnet.radioforus.com](https://mainnet.radioforus.com) |
 | Toko Emas Berlian Harisa | Toko Emas Berlian Harisa, aims to make it easier for pioneers in the world, especially Indonesia, in buying and selling gold and diamonds. | Consumer Utility | | [toko-emas-berlian-harisa](https://github.com/pi-apps/toko-emas-berlian-harisa) | pi://brainstorm.pi/project/6475b0b22c0abb630dcce93d |  [Toko Emas Berlian Harisa Testnet](https://testnet.harisajewellery.com/) | https://harisajewellery.com/ |
+| PyNook | PyNooK is a global barter mall. Through positioning, it can help pi pioneers quickly find merchants who accept pi payment, so as to achieve online and offline fast barter | Javascript/NextJS | [PyNook](https://github.com/picnty/pynook) | [PyNook](https://pynook7269.pinet.com) |
```

Comment:
| PyNook | PyNooK is a global barter mall. Through positioning, it can help pi pioneers quickly find merchants who accept pi payment, so as to achieve online and offline fast barter | Consumer utilities / Business tools | Javascript/NextJS | [PyNook](https://github.com/picnty/pynook)  | [pi://brainstorm.pi/project/65dcae3e6d7f340033b33af4](pi://brainstorm.pi/project/65dcae3e6d7f340033b33af4) | [https://pynook.com](https://pynook.com) | [PyNook Testnet](https://pynook.com) |